client/src/components/Description.jsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
-import {DescriptionContainer, WhyWeMadeThis, ItemDescription} from '../styling.jsx';
+
+const ItemDescription = styled.p`
+  margin: 0px 0px 30px;
+`;
+
+const WhyWeMadeThis = styled.h2`
+  font-size: 1rem;
+  margin: 0px 0px 10px;
```

Comment:
Looks good, but I'm curious, why not margin-bottom?

Reply:
I was tweaking all of the margins and it turns out I only wanted a bottom margin. I could definitely refactor this when I find the time